src/content/docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/nrql-group-results-across-time.mdx

```diff
@@ -24,14 +24,18 @@ With [NRQL](/docs/query-data/nrql-new-relic-query-language/getting-started/intro
 
 <img
   title="NRQL facet by time"
-  alt="NRQL facet by time"
+  alt="A screenshot displaying a NRQL query faceted by time"
   src={queriesFacetByTime}
 />
 
-When using [time functions](/docs/insights/use-insights-ui/time-settings/set-time-range-insights-dashboards-charts) in NRQL queries, the results are returned in UTC. To adjust the results to your time zone, include the [`WITH TIMEZONE` clause](/docs/insights/nrql-new-relic-query-language/nrql-resources/nrql-syntax-components-functions#sel-timezone) in your query.
+When using the time functions from the table below in NRQL queries, the results return in UTC format. To adjust the results to your time zone, include the [`WITH TIMEZONE` clause](/docs/insights/nrql-new-relic-query-language/nrql-resources/nrql-syntax-components-functions#sel-timezone) in your query.
 
 ## Facet your NRQL query time range [#cohorts]
 
+<Callout variant="tip">
+In these examples, we use a custom timestamp attribute submitted with PageView events called `account_created`. To facet by the time of PageView event were ingestion, you could use the `timestamp` attribute instead.
+</Callout>
+
```

Comment:
I'm struggling to sort out where we got the `account_created` reference.  Perhaps the examples have been updated since the ticket was created.  🤷 

Can you replace `account_created` with `createdAt` (to match what's in the examples)?

Reply:
And, change the second sentence to either:
```
To facet by the time of PageView event ingestion, use the `timestamp` attribute instead.
```
or
```
To facet by the time the PageView events were ingested, use the `timestamp` attribute instead.
```

Reply:
Changes made! Thanks very much!